examples/tools/cmake/cmake_toolchain/use_different_toolchain_generator.rst

```diff
@@ -0,0 +1,84 @@
+
+.. _examples-tools-use-different-toolchain-generator:
+
+Using CMakeToolchain with different generators: Ninja example
+=============================================================
+
+This guide demonstrates how to use :ref:`CMakeToolchain<conan_tools_cmaketoolchain>` with predefined generators
+like `Ninja <https://ninja-build.org/>`_ and how to configure it to use different generators.
```

Comment:
Add something like this: This example assumes that Ninja is installed in your system. If it is not, you can use ninja from a Conan package with tool_requires (maybe add an example of this at the end of the example?)

Reply:
@memsharded I added a section about installing Ninja as platform_tool_requires: https://github.com/conan-io/docs/pull/3771/commits/10264d5b0938e370117b950f6118d45b0af26f80

Please, tell me if it's the recommended way.

Reply:
Done, using tool requires instead of platform tool requires.